datawave-micrometer/pom.xml

```diff
@@ -12,8 +12,8 @@
     <artifactId>datawave-micrometer</artifactId>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
```

Comment:
Can we keep it as `17` and just change the compiler target to `11`? 
That would allow it to match the current accumulo build process and future desired datawave build process.